src/wp-admin/includes/schema.php

```diff
@@ -1209,38 +1209,14 @@ function populate_network_meta( $network_id, array $meta = array() ) {
 --The Team @ SITE_NAME'
 	);
 
-	$misc_exts        = array(
-		// Images.
-		'jpg',
-		'jpeg',
-		'png',
-		'gif',
-		'webp',
-		// Video.
-		'mov',
-		'avi',
-		'mpg',
-		'3gp',
-		'3g2',
-		// "audio".
-		'midi',
-		'mid',
-		// Miscellaneous.
-		'pdf',
-		'doc',
-		'ppt',
-		'odt',
-		'pptx',
-		'docx',
-		'pps',
-		'ppsx',
-		'xls',
-		'xlsx',
-		'key',
-	);
-	$audio_exts       = wp_get_audio_extensions();
-	$video_exts       = wp_get_video_extensions();
-	$upload_filetypes = array_unique( array_merge( $misc_exts, $audio_exts, $video_exts ) );
+	$allowed_file_types = array();
+	$all_mime_types     = get_allowed_mime_types();
+
+	foreach ( $all_mime_types as $ext => $mime ) {
+		array_push( $allowed_file_types, ...explode( '|', $ext ) );
+	}
+	sort( $allowed_file_types );
```

Comment:
why are we sorting the allowed mime types?

Reply:
These are not mime types but exts. So example would be `avi, bmp, jpg, mp4`. It makes sense to me keep this in alphabetical  order.

Reply:
that sounds nice but I still don't think we need them sorted :) 

we don't sort file or mime types elsewhere, do we? is there some practical advantage to sorting? if anything I would want the most popular/common types first, so lookups would resolve faster.

Reply:
Alphabetical definitely makes sense, and it's immediately ready for print/view as @spacedmonkey indicates above.